### ISSUE TITLE
Sign up not working 

### ISSUE THREAD

**Issue by Tarekkharsa** [netlify-templates/kpop-stack#101]
when I try to Sign up  after deploy the app I got an error 
<img width="1374" alt="Screen Shot 2022-10-18 at 6 47 00 PM" src="https://user-images.githubusercontent.com/47394961/196480008-b01a7ee6-c844-4041-9aeb-8a25ca231ccf.png">

**Comment by maxcell** [comment]
Just for some context, did you make sure to perform the Supabase Queries so far for setting up the database too? If so, could you deactivate the RLS for me?

**Comment by Tarekkharsa** [comment]
Yes and I try another way to auth I use indie stack and it's the same issue

indie stack: https://tarekkharsa.netlify.app
kpop-stack: https://wondrous-lollipop-d7ed35.netlify.app
I'm sorry What you mean by "deactivate the RLS"

**Comment by ahmed-belhadj** [comment]
I ran into the same issue. Removing double quotes from the env vars fixed it for me because there were extra quotes being passed into the Supabase request.

![Screen Shot](https://user-images.githubusercontent.com/24985688/201501070-a31d09b6-57ea-49a0-81e0-7f6f91d98ab9.png)

**Comment by ricardasjak** [comment]
I ran into same or similar issue, seems that Profile table was not created on my Supabase project by default. And this is an error related to absence of profile data.

```
TypeError: Cannot read properties of null (reading 'id')
    at action5 (/Users/rij/Projects/misc/nebula-remix-n/app/routes/join.tsx:77:18)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by cekuu35** [comment]
@_temp_comment.md
